seq2science/rules/get_fastq.smk

```diff
@@ -60,7 +60,7 @@ rule sra2fastq_SE:
     benchmark:
         expand("{benchmark_dir}/sra2fastq_SE/{{run}}.benchmark.txt", **config)[0]
     wildcard_constraints:
-        run="SRR\d+",
+        run="|".join(sra_single_end) if len(sra_single_end) else "$a",
```

Comment:
this is to prevent snakemake from accusing this rule right? a comment would be nice.